nixos/modules/services/backup/syncoid.nix

```diff
@@ -87,7 +87,7 @@ in
     package = lib.mkPackageOption pkgs "sanoid" {};
 
     interval = lib.mkOption {
-      type = lib.types.str;
+      type = with lib.types; either str (listOf str);
       default = "hourly";
       example = "*-*-* *:15:00";
       description = ''
```

Comment:
Should we update the description to mention what the list means?

Reply:
@SuperSandro2000 Done! Sorry for the delay.

Reply:
Don't worry. You are more impacted by that than we are :)